src/runtimes/node/bundlers/esbuild/plugin_native_modules.ts

```diff
@@ -1,10 +1,9 @@
 import path from 'path'
 
 import type { Plugin } from '@netlify/esbuild'
-import readPackageJson from 'read-package-json-fast'
 
 import { isNativeModule } from '../../utils/detect_native_module.js'
-import { PackageJson } from '../../utils/package_json.js'
+import { PackageJson, readPackageJson } from '../../utils/package_json.js'
```

Comment:
`read-package-json-fast` does `JSON.parse(readFile())` plus some additional validation and checks for binary folders. So our implementation should actually be faster without those checks. :)